OWNERS_ALIASES

```diff
@@ -144,3 +144,8 @@ aliases:
     - xichengliudui
     - zhangxiaoyu-zidif
     - pigletfly
+  sig-docs-pt-owners: # Admins for Portuguese content
+    - femrtnz
+  sig-docs-pt-reviews: # PR reviews for Portugese content
+    - femrtnz
+    - julio.jesus
```

Comment:
I believe this should be `juliojesus` (@juliojesus) ?

I might be wise to add Julio as `sig-docs-pt-owner` as well. Otherwise someone from the English team has to approve your PRs @femrtnz ;) Of course we can do that, but it is definitely good to grant approver privileges to other translators as well ;)

Reply:
100% agreed!